### PR TITLE
Remove activity monitor from web app

### PR DESCRIPTION
**Pull request by rupozzi**
Removing activity monitor following Pull Request [#5969](https://github.com/DIRACGrid/DIRAC/pull/5965).

BEGINRELEASENOTES

CHANGE: removed activity monitor 

ENDRELEASENOTES

### CODE REVIEW

File: src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py [hunk 1]
```diff
@@ -1,257 +0,0 @@
-import os
```
Comment (fstagni) [inline src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py]: This should stay!

Comment (chrisburr) [reply src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py]: It does need modifying though to remove things like: `MonitoringClient().getActivitiesContents`

Comment (rupozzi) [reply src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py]: @chrisburr I don't find that in MonitoringHandler... I think it's activityMonitoringHandler

**Comment by rupozzi** [comment]
Is activityMonitoringHandler.py to be deleted?

**Comment by fstagni** [comment]
> Is activityMonitoringHandler.py to be deleted?

yes, but not with this PR.

**Pull request merged**